### PR TITLE
feat(dashboards): Widget viewer expand and scale big number

### PR DESCRIPTION
**Pull request by edwardgou-sentry**
Expands big number widgets (329k => 329291)
Also scales Big Number according to screen size to better fit smaller screens
image

### CODE REVIEW

File: static/app/components/modals/widgetViewerModal.tsx [hunk 1]
```diff
@@ -198,7 +198,7 @@ function WidgetViewerModal(props: Props) {
     return (
       <React.Fragment>
         {widget.displayType !== DisplayType.TABLE && (
-          <Container>
+          <Container height={widget.displayType !== DisplayType.BIG_NUMBER ? 300 : null}>
```
Comment (priscilawebdev) [inline static/app/components/modals/widgetViewerModal.tsx:201]: should we have the `300` number in a const?

File: static/app/components/modals/widgetViewerModal.tsx [hunk 2]
```diff
@@ -463,8 +464,8 @@ const footerCss = css`
   margin: 0px -${space(4)} -${space(4)};
 `;
 
-const Container = styled('div')`
-  height: 300px;
+const Container = styled('div')<{height?: number | null}>`
+  height: ${p => (defined(p.height) ? `${p.height}px` : 'auto')};
```
Comment (priscilawebdev) [inline static/app/components/modals/widgetViewerModal.tsx:468]: do we need to use the `defined` function here? 
```suggestion
  height: ${p => p.height ? `${p.height}px` : 'auto')};
```

**Comment by priscilawebdev** [comment]
don't we have much space between the number and the table?

image

**Review by priscilawebdev — APPROVED** [review]
left some observations but it looks good to me

File: static/app/components/modals/widgetViewerModal.tsx (hunk 2 above)
Comment (edwardgou-sentry) [reply static/app/components/modals/widgetViewerModal.tsx]: updated!

File: static/app/components/modals/widgetViewerModal.tsx (hunk 1 above)
Comment (edwardgou-sentry) [reply static/app/components/modals/widgetViewerModal.tsx]: Added a constant for the 300!

**Pull request merged**